index.js

```diff
@@ -428,6 +438,18 @@ async function materializeModifiedSchemas(gitRoot = undefined, options = {}) {
     }
 }
 
+function schemaPathToInfoInternal(schema, schemaPath, options = {}) {
```

Comment:
Instead of calling these functions *Internal, maybe we can just indicate that they are 'private' by prefixing them with underscores?

Reply:
Done